### ISSUE TITLE
Mongoid truncation broken in 1.4.0

### ISSUE THREAD

**Issue by iangreenleaf** [DatabaseCleaner/database_cleaner#323]
```ruby
require 'database_cleaner'
DatabaseCleaner[:mongoid].strategy = :truncation
# => DatabaseCleaner::UnknownStrategySpecified: The 'truncation' strategy does not exist for the mongoid ORM!  Available strategies: truncation
```

Seems to be a regression in 1.4.0. The 1.3 series is working fine.

**Comment by thefron** [comment]
:+1: 
I'm experiencing the same issue after updating the gem.

**Comment by steveo7** [comment]
Same here.

**Comment by dhf0820** [comment]
Rolled back to 1.3.0 and works fine. 1.4.0 is broken as above.

**Comment by windy** [comment]
I am just addd ` gem 'mongoid-tree' ` in my Rails Gemfile. it's ok now.
You can try this .

**Comment by DonGiulio** [comment]
adding `gem 'mongoid-tree'` works for me too, if it's a required gem it should be documented. 

thanks

**Comment by awsmsrc** [comment]
just adding a +1 here

**Comment by volodymyr-mykhailyk** [comment]
this bug should be fixed by this commit:
https://github.com/DatabaseCleaner/database_cleaner/commit/88479b71c9b4b82872fc037d54a26128edab6b7f

**Comment by mindreframer** [comment]
please release a new version without the dependency on mongoid-tree, it's been 2 months already since this issue is known. 

thx!

**Comment by vitalis** [comment]
:+1:

**Comment by lucke84** [comment]
I believe this is the same as https://github.com/DatabaseCleaner/database_cleaner/issues/299, where @etagwerker just promised a new release. Maybe this one can be closed.

**Comment by AknEp** [comment]
This issue looks to be solved (latest version; 1.4.1). Why don't you close it? @iangreenleaf

**Comment by satoryu** [comment]
or @etagwerker could you please close this issue?
